Index.html

```diff
@@ -0,0 +1,200 @@
+                src="https://img.icons8.com/color/100/000000/ballerina-full-body.png"
+                alt="logo"
+              />
+              <p>
+                Nobody cares if you can’t dance well. Just get up and dance.
+                Great dancers are great because of their passion.
+              </p>
+            </li>
+            <li>
+              <img
+                src="https://img.icons8.com/color/96/000000/dancing-party.png"
+                alt="logo"
+              />
+              <p>
+                To dance is to be out of yourself. Larger, more beautiful, more
+                powerful… This is power, it is glory on earth and it is yours
+                for the taking.
+              </p>
+            </li>
+            <li>
+              <img
+                src="https://img.icons8.com/external-photo3ideastudio-lineal-color-photo3ideastudio/100/000000/external-dancer-celebration-photo3ideastudio-lineal-color-photo3ideastudio.png"
+                alt="logo"
+              />
+              <p>
+                Those who were seen dancing were thought to be insane by those
+                who could not hear the music.
+              </p>
+            </li>
+            <li>
+              <img
+                src="https://img.icons8.com/officel/80/000000/tango.png"
+                alt="logo"
+              />
+              <p>
+                “Hand in hand, on the edge of the sand, they danced by the light
+                of the moon.”
+              </p>
+            </li>
+            <li>
+              <img
+                src="https://img.icons8.com/dusk/100/000000/dancing.png"
+                alt="logo"
+              />
+              <p>
+                Dance for yourself. If someone understands, good. If not, no
+                matter.
+              </p>
+            </li>
+          </ul>
+          <div class="program-button-container">
+            <button type="submit" class="program-button"><span>See More</span></button>
+          </div>
+        </div>
+        <div class="program-link">
+          <a href="#">See More</a>
+        </div>
+      </section>
+      <section class="feature-section">
+        <h2>Best Dancer's</h2>
+        <ul class="feature">
+        </ul>
+       <div class="showcontainer">
+        <div class="showMore">
+         <span> More</span>
+        </div>
+       </div>
+      </section>
+      <section class="partners">
+        <h2>Partners</h2>
+        <div class="partner">
+          <div class="partner-image">
+            <img src="./image/BAFTA_logo.svg" alt="logo" />
+          </div>
+          <div class="partner-image">
+            <img src="./image/Disney_Channel_logo.svg" alt="logo" />
+          </div>
+          <div class="partner-image">
+            <img src="./image/Michael_Kors_logo.svg" alt="logo" />
+          </div>
+          <div class="partner-image">
+            <img src="./image/Zapier_logo.svg" alt="logo" />
```

Comment:
- [ ] Kindly, update the `background-color` of the `partners` section to match the design.